modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpi.java

```diff
@@ -382,7 +386,10 @@
     /** Default SO_LINGER to use for socket. Set negative to disable, non-negative to enable, default is {@DFLT_SO_LINGER }. */
     private int soLinger = DFLT_SO_LINGER;
 
-    /** IP finder clean frequency. */
+    /**
+     * The frequency with which coordinator cleans IP finder and keeps it in the correct state, unregistering addresses of the
```

Comment:
unregister_s

Reply:
Maybe "unregistered"

Reply:
Let's rewrite it in an easier way, "The frequency with which coordinator cleans IP finder and keeps it in the correct state, which means that coordinator unregisters addresses of the nodes that have left the topology and re-registries missing addresses."  Also, it might be helpful to write a link to `org.apache.ignite.spi.discovery.tcp.ServerImpl.IpFinderCleaner#cleanIpFinder` in javadoc.

Also it would be good to fix javadoc for `org.apache.ignite.spi.discovery.tcp.ServerImpl.IpFinderCleaner`